frame/contracts/src/lib.rs

```diff
@@ -781,113 +794,3 @@ impl<T: Trait> Config<T> {
 		T::Currency::minimum_balance().saturating_add(T::TombstoneDeposit::get())
 	}
 }
-
-/// Definition of the cost schedule and other parameterizations for wasm vm.
```

Comment:
fwiw I am super up for removing stuff from `lib.rs`

Reply:
Me too.